src/main/java/de/muenchen/isi/api/dto/abfrageInBearbeitungSachbearbeitung/AbfrageInBearbeitungSachbearbeitungDto.java

```diff
@@ -58,4 +59,7 @@ public abstract class AbfrageInBearbeitungSachbearbeitungDto {
     private ArtAbfrage artAbfrage;
 
     private UUID bauvorhaben;
+
+    @Size(max = 255, message = "Es sind maximal {max} Zeichen erlaubt")
+    private String linkEakte;
```

Comment:
Da es sich bei URIs in der EAkte um Deeplinks handelt können diese sehr lange sein.

Die Länge sollte auf mindestens 8000 Octets (characters) erhöht werden um dem folgenden RFC zu entsprechen.

https://www.rfc-editor.org/rfc/rfc9110#section-4.1-5